ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -2194,7 +2194,7 @@ public sealed class Bot : IAsyncDisposable {
 
 	private static async Task LimitLoginRequestsAsync() {
 		if ((ASF.LoginSemaphore == null) || (ASF.LoginRateLimitingSemaphore == null)) {
-			ASF.ArchiLogger.LogNullError($"{nameof(ASF.LoginSemaphore)} || {nameof(ASF.LoginRateLimitingSemaphore)}");
+			ASF.ArchiLogger.LogNullError(null, $"{nameof(ASF.LoginSemaphore)} || {nameof(ASF.LoginRateLimitingSemaphore)}");
```

Comment:
We must change it then, I suggest splitting into two `if` checks and two standalone log nulls.